includes/class-wc-stripe-webhook-handler.php

```diff
@@ -845,7 +845,8 @@ public function process_payment_intent_success( $notification ) {
 				break;
 			case 'payment_intent.succeeded':
 			case 'payment_intent.amount_capturable_updated':
-				$charge = end( $intent->charges->data );
```

Comment:
This issue should only affect webhooks, because we're overwriting the API version [here](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/1833465bc14d3a699abec9ec7d4f64b0fbdb4f24/includes/class-wc-stripe-api.php#L17) when we make the requests ourselves.

When we update the plugin to the latest API, then we would want to add this and other changes.